docs/docs/01-introduction.md

```diff
@@ -54,15 +56,16 @@
 - Babel
 - TypeScript
 - PostCSS
-- SASS
+- Sass
```

Comment:
✨

Reply:
Haha a hill worth dying on?

Reply:
Absolutely

That and smart quotes/em-dashes (which I might to a F&R on at some point 😛)